File: cexercicios/cadastro_cliente.c
/*
    Problema: Crie um algoritmo que leia para um cadastro de cliente, 
    o seu nome, sua idade, o seu salário e mostre o resultado.

    Autor: José Brenon - 06/11/2023 
*/
#include <stdio.h>
struct cadastro
{
    char nome [30];
    int idade;
    float salario;
};
int main()
{   
    struct cadastro dados_cliente;
    printf("Digite o nome do cliente: ");
    scanf("%s", dados_cliente.nome);
    printf("Digite a idade do cliente: ");
    scanf("%d", &dados_cliente.idade);
    printf("Digite o salario do cliente: ");
    scanf("%f", &dados_cliente.salario);

    printf("\nO nome do cliente: %s",dados_cliente.nome);
    printf("\nA idade do cliente: %d",dados_cliente.idade);
    printf("\nO salário do cliente: %f",dados_cliente.salario);
    printf("\n\n\n............FIM..........\n");
    return 0;
}

File: javaexercicios/cadastro_cliente.java
/*
    Problema: Crie um algoritmo que leia para um cadastro de cliente, 
    o seu nome, sua idade, o seu salário e mostre o resultado.

    Autor: José Brenon - 06/11/2023 
*/

import javax.swing.JOptionPane;

public class cadastro_cliente {
    public static void main(String[] args) {
        cadastro dados_cliente = new cadastro();

        dados_cliente.nome = JOptionPane.showInputDialog("Digite o nome do cliente");
        dados_cliente.idade = Integer.parseInt(JOptionPane.showInputDialog("Digite a idade do cliente"));
        dados_cliente.salario = Float.parseFloat(JOptionPane.showInputDialog("Digite o salário do cliente"));
        
        JOptionPane.showMessageDialog(null, "O nome do cliente é: "+dados_cliente.nome+"\nA idade do cliente é: "+dados_cliente.idade+"\nO salário do cliente é: "+dados_cliente.salario);
    }

}
class cadastro {
    String nome;
    int idade;
    float salario;
}